src/reflect.js

```diff
@@ -63,3 +63,8 @@ if (typeof Reflect.construct !== 'function') {
     return new (bind.apply(Target, a));
   };
 }
+
+if (typeof Reflect.ownKeys !== 'function')
+    Reflect.ownKeys = function (o) {
+    return (Object.getOwnPropertyNames(o).concat(Object.getOwnPropertySymbols(o))));
```

Comment:
My bad, I didn't notice Object.getOwnPropertySymbols is already included in aurelia-polyfills.